src/Core/Checkout/Document/Extension/HtmlRendererExtension.php

```diff
@@ -0,0 +1,33 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Checkout\Document\Extension;
+
+use Shopware\Core\Checkout\Document\Renderer\RenderedDocument;
+use Shopware\Core\Framework\Extensions\Extension;
+use Shopware\Core\Framework\Log\Package;
+
+/**
+ * @public this class is used as type-hint for all event listeners, so the class string is "public consumable" API
+ *
+ * @title Rendering of the PDF document
```

Comment:
when you copy and paste code with doc comments, please adjust the docs accordingly ;)